src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,92 @@
+package seedu.addressbook.ui;
+
+import static seedu.addressbook.common.Messages.MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE;
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+import static seedu.addressbook.common.Messages.MESSAGE_WELCOME;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Formatter {
+	
+	/** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    public String enterCommand(){
+    	return LINE_PREFIX+"Enter command: ";
+    }
+    
+    public String[] showWelcomeMsg(String version, String storageFilePath){
+    	String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
+    	String[] welcomeMsg=new String[7];
+    	welcomeMsg[0]=DIVIDER;
+    	welcomeMsg[1]=DIVIDER;
+    	welcomeMsg[2]=MESSAGE_WELCOME;
+    	welcomeMsg[3]=version;
+    	welcomeMsg[4]=MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE;
+    	welcomeMsg[5]=storageFileInfo;
+    	welcomeMsg[6]=DIVIDER;
+    	return welcomeMsg;
+    }
+    
+    public String formatString(String message){
+    	return LINE_PREFIX + message.replace("\n", LS + LINE_PREFIX);
+    }
+    
+    public String formatOtherMessage(String otherMessage){
```

Comment:
Again, consider a better name. User of this class will be confused by "OtherMessage" and wonder what does it mean.